skyfield/documentation/positions.rst

```diff
@@ -98,6 +99,10 @@ as its argument and return a corresponding number of positions.
     astrometric = boston(jd).observe(mars)
     apparent = boston(jd).observe(mars).apparent()
 
+    # Earth location where in zenith at date
+
+    topos = apparent.over_location()
```

Comment:
Wow! You return a new `topos` object! That would never had occurred to me — I just would have returned a `lat, lon` pair from a `sublatlon` function or something, which would have been a dead end because you can't ask a bare `lat` and `lon` further questions like you can use a `Topos` to follow up. I think I like the idea, and I think we will definitely keep this approach!